packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1095,6 +1095,936 @@ curl -v -X DELETE \
 HTTP/1.1 204 No Content
 ```
 
+## Email template operations
+
+The Email Templates API has the following operations:
```

Comment:
I recommend removing this line for a few reasons:
- it's unnecessary
- the Email Templates operations are not an API
- removing it follow the format of many of the other pages

```suggestion
```